api/items_post.md

```diff
@@ -80,6 +80,41 @@ understands. The meaning of each key is explained in the comments.
       // If a message with no stack trace, use "message"
       // If an iOS crash report, use "crash_report"
 
+      // Optional: "telemetry". Only applicable if you are sending [telemetry data](/docs/notifier/rollbar.js/#telemetry). 
+      "telemetry": {
+        // Required: level
+        // The severity level of the telemetry data. One of: "critical", "error", "warning", "info", "debug".
+        "level": "info",
+
+        // Required: type
+        // The type of telemetry data. One of: "log", "network", "dom", "navigation", "error", "custom".
```

Comment:
`custom` is not a type is it? Rollbar.js uses `manual` for the type of manual generated events

Reply:
@coryvirok or @rokob can you help me with this?

Reply:
Looks like `manual` is indeed used on the frontend however that appears to be a bug based on the techspec. 

@chrisbarmonde can we add `custom` as an alias to `manual`? I think the only place that checks/uses it is in the react code.

https://github.com/rollbar/mox/blob/c069ed3880593b65f7845bcba17fc0fca8ba4d3a/mox/frontend/js/item/telemetry/Breadcrumb.js#L14

Reply:
It kinda sounds like what we have now is working fine and it's just the techspec that isn't accurate? While I started off using the spec, I ultimately just followed along with what was happening in the notifier which is why the notifier and the frontend align right now.

Reply:
Ah, I didn't realize rollbar.js also used 'manual', as Andrew mentioned. (https://github.com/rollbar/rollbar.js/blob/5d3c47f1bb2e944d35b7107e822ab6e2ebab0ac2/src/telemetry.js#L44)

@rivkahstandig3636 let's just change custom -> manual and call it a day.